charts/datahub/subcharts/datahub-frontend/templates/deployment.yaml

```diff
@@ -15,10 +15,13 @@ spec:
       {{- include "datahub-frontend.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-    {{- with .Values.podAnnotations }}
       annotations:
+      {{- with .Values.podAnnotations }}
         {{- toYaml . | nindent 8 }}
-    {{- end }}
+      {{- end }}
+      {{- if .Values.defaultUserCredentials }}
+        checksum/secret: {{ include (print $.Template.BasePath "/user-secrets.yaml") . | sha256sum }}
```

Comment:
This will change every time if `randomAdminPassword` is enabled. Should check for `manualValues` too?

Reply:
To me, that makes sense even if the `randomAdminPassword` is used, because it ensures that the password used by the Pod is always in sync with the secret. It is very confusing otherwise to try looking up credentials in a secret and then not being able to use them because the pod has an outdated value of them.